src/cavendish_particle_tracks/analysis.py

```diff
@@ -179,5 +189,12 @@ def calibrate(self) -> None:
     def to_csv(self):
         mystring = ""
         for var in self.vars_to_save():
-            mystring += str(getattr(self, var)) + ","
+            if var in ["rpoints", "dpoints"]:
+                mystring += "["
+                for point in getattr(self, var):
+                    x, y = point
+                    mystring += "[{x} {y}]; "
```

Comment:
```suggestion
                    mystring += "f[{x} {y}]; "
```

Reply:
Sorry. Would be good maybe to commit an expected CSV file into test data? Can then do regression testing.

Reply:
Looks like [`filecmp`](https://docs.python.org/3/library/filecmp.html) does it. And it's a builtin.

- https://stackoverflow.com/q/75838502/1444054
   (the first comment about asserting on an empty diff looks nice).